hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/TestHddsDatanodeService.java

```diff
@@ -50,6 +54,9 @@ public void setUp() {
     conf.set(HddsConfigKeys.OZONE_METADATA_DIRS, testDir.getPath());
     conf.setClass(OzoneConfigKeys.HDDS_DATANODE_PLUGINS_KEY, MockService.class,
         ServicePlugin.class);
+    conf.setBoolean(OZONE_SECURITY_ENABLED_KEY, false);
```

Comment:
by default security is false right?

Reply:
It's ok to set here again for better understanding the situation. Can you add a comment, why we set them this way?
Just include JIRA ID also. Thanks.